Packs/CommonScripts/Scripts/diplayMappedFields/diplayMappedFields.py

```diff
@@ -14,50 +21,30 @@ def extract_keys_with_values(obj, parent_key=''):
     return items
 
 
-def format_data_to_table(items):
-    """Formats the extracted data into a table format"""
-    table = '| Field Name | Value |\n'
-    for key, value in items:
-        table += f'| {key} | {value} |\n'
-    return table
-
-
-def convert_to_html(markdown):
-    html = "<table style='border-collapse: collapse;'>"
-    lines = markdown.split("\n")
-    for line in lines[1:]:
-        html += "<tr>"
-        columns = line.split("|")
-        for i, column in enumerate(columns):
-            if column.strip():
-                if i != 1:
-                    html += f"<td style='font-family: Lato, Assistant, sans-serif; font-weight: 600; \
-                    font-size: 12px; text-align: left;  \
-                    padding: 1px 0px 0px; margin: 0px 5px 0px 0px; contrast: 4.95; ֿֿֿ\
-                    color: var(--xdr-on-background) ;'>{column.strip()}</td>"
-                else:
-                    html += f"<td style='font-family: Lato, Assistant, sans-serif; font-weight: 600; \
-                    font-size: 12px; text-align: left; \
-                    padding: 1px 0px 0px; margin: 0px 5px 0px 0px; contrast: 4.95; \
-                    color: var(--xdr-on-background-secondary);'>{column.strip()}</td>"
-        html += "</tr>"
-    html += "</table>"
-    return html
-
-
-def remove_empty_rows(table):
-    # Split the table into rows
-    rows = table.split("\n")
+def format_data_to_rows(items):
+    """
+    Formats the extracted data into rows.
+    """
+    return [f'{key}|{value}' for key, value in items]
 
-    # Filter out the rows that contain empty dictionaries
-    filtered_rows = [row for row in rows if not (
-        "{}" in row or "[{}]" in row or "[{}, {}]" in row or "[{}, {}, {}]" in row or "0001-01-01T00:00:00Z" in row or " \
-        containmentsla" in row or "remediationsla" in row or "detectionsla" in row or "triagesla" in row)]
 
-    # Join the filtered rows back into a table
-    filtered_table = "\n".join(filtered_rows)
+def convert_to_html(rows):
+    html = ["""<table style="border-collapse:collapse;"><tbody style="font-family:Lato,Assistant,sans-serif;font-weight:600;font-size:12px;text-align:left;padding: 1px 0px 0px;margin:0px 5px 0px 0px;contrast:4.95">"""]  # noqa: E501
```

Comment:
the bug fix: moved the html tags to the `<tbody>` to reduce the output. the rendering remains the same.

Reply:
Using CSS and classes would have saved even more bytes... :-)

Reply:
backend platform sanitized any CSS styling or html `<style>` tags and other type of CSS hacks,  which isn't embedded like what i did.